wiki/CreatingInterfaces.md

```diff
@@ -242,11 +245,14 @@ If `color` is defined but either the inactive or hover color is not, the undefin
 		from <x#> <y#> to <x#> <y#> [<anchor>]
 		color <color>
 		size <size#>
+		[reversed]
+		[start angle <angle#>]
+		[span angle <angle#>>]
 ```
 Defines a straight line (bar) or circular outline (ring) to be drawn with this interface.
-In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box.
+In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box. The start and span angle specifies the portion of the ring that is drawn **(v. 0.10.2)**.
```

Comment:
New line?
Also, some more details would be good.
```suggestion
In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box.
Beginning in **v. 0.10.2**, it is also possible to define the "start angle" and "span angle" for a ring.
The "start angle" defines where along the radius filling of the ring should begin. The default value is 0, which corresponds to straight up. [Also mention which direction it goes in.]
[I'll need to go check the original PR to produce a better description of span angle. I'll fill these in later.]
```

Reply:
@tibetiroka updated description above.

Reply:
> Values as low as 0 are allowed.

"The minimum value is 0, in which case nothing is drawn."

Reply:
Done